docs/recipes/Deploying-Modules.md

```diff
@@ -5,83 +5,23 @@
 # Deploying Modules
 
 ## 📖 Table of Contents
+* [Deployment Overview](deployment-overview)
+* [Creating a Module Map](#creating-a-module-map)
+* [Update Module Map Script](#update-module-map-script)
+* [Deploying to Vercel with GitHub Actions](#deploying-to-vercel-with-github-actions)
 
-* [Deploying to Now with GitHub Actions](#deploying-to-now-with-github-actions)
-  * [Creating your Deploy Action](#creating-your-deploy-action)
-* [Create and Update Your Module Map](#create-and-update-your-module-map)
-  * [Creating a Module Map in Now.sh](#creating-a-module-map-in-nowsh)
-  * [Create an Update Module Script](#create-an-update-module-script)
-  * [Update your Deploy Action to Update your Module Map](#update-your-deploy-action-to-update-your-module-map)
+## Deployment Overview
 
-## Deploying to Now with GitHub Actions
+Deploying holocron modules consists on two steps: 
 
-[Now.sh](https://zeit.co/home) allows you to deploy statics such as your modules. We will accomplish this by using [GitHub Actions](https://github.com/features/actions).
+1. Upload the contents of the `build` folder to your CDN of choice.
```

Comment:
I think I used to have that structure in my original script, but guess for Vercel was removed because it looks like you can't upload more than 1 module to the same project/url so every module will have it's own CDN instance which is a pain. With Amazon S3 you can keep all your modules under the same url/project so it the folder is required there

Reply:
Added a note that vercel doesn't support multiple folders